egui-d3d9/src/state.rs

```diff
@@ -54,7 +51,8 @@ impl DxState {
             let mut original_proj: Matrix4x4 = Default::default();
 
             expect!(
-                dev.GetTransform(D3DTS_WORLD, &mut original_world),
+                // https://github.com/apitrace/dxsdk/blob/d964b66467aaa734edbc24326da8119f5f063dd3/Include/d3d9types.h#L333C35-L333C56
+                dev.GetTransform(D3DTRANSFORMSTATETYPE(0 + 256), &mut original_world),
```

Comment:
https://learn.microsoft.com/en-us/windows/win32/direct3d9/d3dts-worldmatrix

For some reason the D3DTS_WORLD macro is gone on the new version?

https://learn.microsoft.com/en-us/windows/win32/direct3d9/d3dts-world